dev-infrastructure/docs/development-setup.md

```diff
@@ -357,10 +376,10 @@ make db/setup
 ./clusters-service init --config-file ./development.yml
 ```
 
-4) Start CS:
+5) Start CS:
 
 ```bash
-./clusters-service serve --config-file development.yml --runtime-mode aro-hcp --azure-auth-config-path azure-creds.json
+./clusters-service serve --config-file development.yml --runtime-mode aro-hcp --azure-auth-config-path azure-creds.json --azure-runtime-config-path ./azure-runtime-config.json --azure-first-party-application-certificate-bundle-path ~/fpa_cert_decoded --azure-first-party-application-client-id $CLIENT_ID
```

Comment:
All of these flags are in the development.yml file so you don't have to specify it as separate flags: https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/blob/master/configs/development.yml?ref_type=heads#L246-259